src/engine.cc

```diff
@@ -52,6 +52,7 @@ const char* kMoveOverheadStr = "Move time overhead in milliseconds";
 const char* kTimeCurvePeak = "Time weight curve peak ply";
 const char* kTimeCurveRightWidth = "Time weight curve width right of peak";
 const char* kTimeCurveLeftWidth = "Time weight curve width left of peak";
+const char* kSyzygyTablebaseStr = "Syzygy tablebase paths";
```

Comment:
Because it's a colon separated list of directories to search in (or semi colon on windows)

Reply:
Maybe something like `List of syzygy tablebase directories" so that it's more descriptive help message?

Also when Arena sees some words in description, it does special UI (e.g. if there is "file", there is a button for file selection). If arena supports directory selection, would be nice to trigger that too.